### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by SheronPerera02**

### CODE REVIEW

**Comment by dulajdilshan** [comment]
Is this after refactoring? @SheronPerera02

**Comment by SheronPerera02** [comment]
> Is this after refactoring? @SheronPerera02

I cleaned up the code. Whats left are what we need for further development.

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    //     exitFullscreen: "Exit fullscreen",
+    //     frameTitle: "Player for {title}",
+    //     captions: "Captions",
+    //     settings: "Settings",
+    //     menuBack: "Go back to previous menu",
+    //     speed: "Speed",
+    //     normal: "Normal",
+    //     quality: "Quality",
+    //     loop: "Loop"
+    // }
```
Comment (dulajdilshan) [inline src/App.js]: Why we have commented this?

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    };
+  }, []);
+
+  return (
+    <div className={classes.ParentContainer}>
+      <div className={classes.VideoContainer}>
+        <div id="player">
+          <iframe
+            title="My Video"
+            // src="https://www.youtube.com/watch?v=bTqVqk7FSmY"
```
Comment (dulajdilshan) [inline src/App.js]: Is this a testing video?

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            src="https://www.youtube-nocookie.com/embed/bTqVqk7FSmY"
+            // src={`https://www.youtube.com/embed/${'bTqVqk7FSmY'}?showinfo=0&enablejsapi=1&origin=http://localhost:3000`}
```
Comment (dulajdilshan) [inline src/App.js]: This is also commented?

File: src/App.js (hunk 1 above)
Comment (SheronPerera02) [reply src/App.js]: Kept it for reference; what functions i could use on the player.

File: src/App.js (hunk 2 above)
Comment (SheronPerera02) [reply src/App.js]: Yes

File: src/App.js (hunk 3 above)
Comment (SheronPerera02) [reply src/App.js]: I'll clean this up even further

**Pull request merged**